.changeset/seven-roses-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@svelte-add/config': patch
+---
+
+chore: move `mdsvex` to a different category
```

Comment:
will need to also add `svelte-add` here too

Reply:
could we make one a peer dep of the other or something to automatically trigger adding svelte-add when config is updated?

Reply:
how about fixing them together via changesets instead: https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md

this seems like it's what we're looking for

Reply:
As per the bot above it's intending to release that package as well based on that singular changeset

Reply:
now i'm confused lol - why is it now incrementing the versions of dependents when it doesn't do that for `@svelte-add/adders`?

Reply:
the only difference that I can see is that `@svelte-add/adders` is a private package, but that's about it?

Reply:
It's because `config` is a dependency, and `adders` are not. `adders` are just dynamically imported, if i recall correctly.

Reply:
ah, yea you're right. it's cause `@svelte-add/adders` is a devDependency rather than a dependency, so it doesn't increment. very interesting